src/main/java/io/reactivex/Flowable.java

```diff
@@ -14668,7 +14668,7 @@ public final void subscribe(Subscriber<? super T> s) {
     @BackpressureSupport(BackpressureKind.UNBOUNDED_IN)
     @SchedulerSupport(SchedulerSupport.NONE)
     public final <K> Single<Map<K, Collection<T>>> toMultimap(Function<? super T, ? extends K> keySelector) {
-        Function<T, T> valueSelector = Functions.identity();
+        Function<? super T, ? extends T> valueSelector = Functions.identity();
```

Comment:
Such changes in our own source is a good indication there will be generics issues for the users of the library.